scanner/templates/django/Dockerfile

```diff
@@ -49,6 +49,8 @@ EXPOSE 8000
 
 {{ if .wsgiFound -}}
 CMD ["gunicorn", "--bind", ":8000", "--workers", "2", "{{ .wsgiName }}.wsgi"]
+{{ else if .asgiFound -}}
+CMD ["daphne", "-b", "0.0.0.0", "-p", "8000", "{{ .asgiName }}.asgi:application"]
```

Comment:
@felixxm should we also check if either `gunicorn` or `daphne` (for example) are in the `requirements.txt`? In case we find both `wsgi.py` and `asgi.py` files, it will default to `gunicorn` here.

Reply:
Good idea, I added check for an installed web server.